### ISSUE TITLE
Pod resources needs fine-tuning

### ISSUE THREAD

**Issue by howardjohn** [istio/test-infra#1457]
Currently most of our jobs are set to 3CPU request/limit. We set them to the same in hopes of preventing our nodes from getting maxed out which we thought was causing issues -- we are not 100% sure if it is the cause or not. Memory is generally set to 3gb/24gb limit.

List of jobs I have seen exceeding these:

* integ-security-k8s-presubmit-tests-master using 4.2GB
* Linter using 18gb when it requests 16gb
* racetest using 8GB vs 3GB request
* codecov using 5GB vs 3gb request
* Basically any job will get CPU throttled during the start of the test when it is building images.
... more TBD ...

We should also get some automation around this, likely dashboards, to record test resource utilization

**Comment by howardjohn** [comment]
release-test using 4.5GB memory

**Comment by howardjohn** [comment]
This is more or less done
